test/AutoDiff/SILGen/vtable.swift

```diff
@@ -1,4 +1,6 @@
-// RUN: %target-swift-frontend -emit-silgen %s -requirement-machine=off | %FileCheck %s
+// RUN: %target-swift-frontend -emit-silgen %s | %FileCheck %s
+
+// FIXME(rdar://84987079)
```

Comment:
What is this FIXME in relation to? The test appears to be enabled

Reply:
This was accidental. I deleted it.